### PR TITLE
TAJO-1509 Use dedicated thread to release resource allocated to container

### PR DESCRIPTION
**Pull request by navis**

### CODE REVIEW

**Comment by navis** [comment]
Can anyone review this?

**Comment by babokim** [comment]
@navis I'll review this patch tonight.

File: tajo-core/src/main/java/org/apache/tajo/worker/TajoResourceAllocator.java [hunk 1]
```diff
@@ -68,6 +73,7 @@
   private TajoConf tajoConf;
   private QueryMasterTask.QueryMasterTaskContext queryTaskContext;
   private final ExecutorService executorService;
```
Comment (babokim) [inline tajo-core/src/main/java/org/apache/tajo/worker/TajoResourceAllocator.java]: "allocationExecutor" is to be more appropriate than "executorService".

**Comment by babokim** [comment]
@navis, thank you for your contribution. This patch looks good to me except one trivial variable name concern.

**Comment by babokim** [comment]
@navis I think you may very busy to resolve other issues. If you don't mind I will reflect what I commented before committing.

**Comment by navis** [comment]
Sure, I'm busy on learning the greatness of SK group till next week.

2015년 4월 13일 월요일, Hyoungjun Kim<notifications@github.com>님이 작성한 메시지:

**Comment by jihoonson** [comment]
@babokim, it seems that the commit log does not include who committed this patch. Please add it from the next commit.

**Comment by babokim** [comment]
@jihoonson Do you mean I should use "--signoff" git option when commit?

**Comment by jihoonson** [comment]
I think it doesn't matter how you record it. You can use the ```--signoff``` option or just write by hand in the log.

**Pull request closed without merging**